packages/carbon/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": ["../../.eslintrc-typescript"],
+  "plugins": ["@typescript-eslint", "jsx-a11y", "react", "import", "@emotion"],
+  "rules": { "@emotion/jsx-import": "error" }
```

Comment:
I'm guessing that carbon uses emotion?

Reply:
Carbon doesn't use emotion, I copy most settings from @rjsf/chakra-ui. Going to remove code about emotion.